app/src/main/java/decide/lic/LIC8.java

```diff
@@ -0,0 +1,40 @@
+package decide.lic;
+
+import decide.Functions;
+
+import decide.datastructures.Parameters;
+import decide.datastructures.Point;
+
+public class LIC8 extends AbstractLIC {
+    Functions functions = new Functions();
+
+    @Override
+    public boolean compute(Point[] points, Parameters parameters) {
+        
+        if (points.length<5) {
+            return false;
+        }
+
+        int totalPointsThatShouldBeConsecutive = 3 + parameters.A_PTS + parameters.B_PTS;
+
+        System.out.println("Test with radius: "+parameters.RADIUS1);
+
+        for (int i = 0; i <= points.length - totalPointsThatShouldBeConsecutive; i++) {
+            int firstIndex = i;
+            int secondIndex = firstIndex + parameters.A_PTS + 1;
+            int thirdIndex = secondIndex + parameters.B_PTS + 1 ; 
+            System.out.println("indexes: "+ firstIndex + " - "+secondIndex+" - "+thirdIndex + " Radius: "+parameters.RADIUS1);
+            Point circlesCenter = functions.getMidPointOfTheLongestSideInTriangle(points[firstIndex], points[secondIndex], points[thirdIndex]);
+            Point pointOutsideCircle = functions.getPointOusideOfTheLongestSideInTriangle(points[firstIndex], points[secondIndex], points[thirdIndex]);
+            if(functions.isPointOutsideCircle(circlesCenter, parameters.RADIUS1, pointOutsideCircle)){
+                System.out.println("returned true");
+                return true;
```

Comment:
The center of the circumscribed circle is not necessary the mid point of the longest side of the triangle. You can for example consider the equilateral triangle of side length 1. If you consider your code with RADIUS1 = 0.6, the function will return true but shouldn't.

Indeed, for this triangle, the three points are on the circumscribed circle which radius is 1/sqrt(3) = 0.577. So the function should return false as the vertices can be contained within a circle of radius RADIUS1.

If you take the center of one of the sides, then you will need a circle of radius sqrt(3)/2 = 0.866 (height of the triangle) to contain the three vertices. The function then returns true.

Reply:
Thanks, made changes now.